pkg/storage/s3/s3.go

```diff
@@ -439,6 +439,7 @@ func (d *driver) RemoveStorage(cr *opapi.ImageRegistry, modified *bool) error {
 	cr.Spec.Storage.S3.Bucket = ""
 	d.Config.Bucket = ""
 	cr.Status.Storage.S3 = d.Config
+	*modified = true
```

Comment:
no.  you need to actually check if you are changing the value.  if you do this, you're creating hotloops again.